modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -5,16 +5,20 @@
 [id="metering-store-data-in-shared-volumes_{context}"]
 = Storing data in shared volumes
 
+Metering has no storage by default, but it can use any ReadWriteMany persistent volume (PV) or any storage class that provisions a ReadWriteMany PV.
```

Comment:
How do you feel about `Metering configures no storage by default, ...`

I don't have a strong opinion either way.

Reply:
@timflannagan Updated to the following if the wording reads the same:

"Metering does not configure storage by default. However, you can use any ReadWriteMany persistent volume (PV) or any storage class that provisions a ReadWriteMany PV for Metering storage."